starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,91 @@
 // Names and Input
 
+var hacker1 = "John"
+
+console.log("The driver's name is " + hacker1);
+
+var hacker2 = prompt("What is your name?");
+
+console.log("The navigator's name is " + hacker2);
 
 //Conditionals
 
+if(hacker1.length > hacker2.length){
```

Comment:
Try to be consistent with the format, I would put a space between the " ) " of the condition and the " { ". Same for the followings.